.gitignore

```diff
@@ -20,3 +20,4 @@ OALLexer.tokens
 .externalToolBuilders
 /test/plugin/dist
 /test/plugin/workspace
+e2e-*
```

Comment:
What is this for?

Reply:
Local e2e will generate related files, reduce error commits.